_patterns/charts_graphs.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: default
+type: component
+title: Charts and Graphs
```

Comment:
This has been moved out of our Sprint priorities, so I think this can be removed for now.